mkdocs.yml

```diff
@@ -91,8 +91,7 @@ extra:
     link: https://discord.gg/5wjP76mBJD
   - icon: fontawesome/brands/stack-overflow
     link: https://stackoverflow.com/tags/ddev
-  - icon: fontawesome/brands/twitter
-    link: https://twitter.com/search?q=%23ddev&src=unknown&f=live
+
```

Comment:
This might leave a gap on the footer though. As the icon might be missing.

Reply:
It's rendered at https://ddev--6799.org.readthedocs.build/en/6799/ and I don't see a gap, but I don't particularly like the result. Not sure if this is an artifact of the PR build.

Reply:
![Screenshot 2024-12-02 at 8 13 40 PM](https://github.com/user-attachments/assets/b7141686-7b32-480a-87c0-17476f9c2cba)
I guess that deletes it. I thought we would want Bluesky down there too but it might not be needed.